DOCUMENTATION.md

```diff
@@ -6,12 +6,12 @@ This documentation is for developers interested in using the GOV.UK Notify PHP c
 
 The Notify PHP Client is based on a [PSR-7 HTTP model](https://www.php-fig.org/psr/psr-7/) [external link]. To install it, you must select your preferred PSR-7 compatible HTTP client. You can follow these instructions to use [Guzzle v6 and v5](http://docs.guzzlephp.org/en/stable/)[external link].
```

Comment:
There is a reference to Guzzle v6 and v5 in this paragraph that should probably be removed?